src/lib/markbind/src/lib/markdown-it/index.js

```diff
@@ -109,8 +109,10 @@ markdownIt.renderer.rules.fence = (tokens, idx, options, env, slf) => {
   const heading = token.attrGet('heading');
   const codeBlockContent = `<pre><code ${slf.renderAttrs(token)}>${str}</code></pre>`;
   if (heading) {
-    return '<div class="code-block">'
-      + `<div class="code-block-heading"><span>${heading}</span></div>`
+    const renderedHeading = markdownIt.renderInline(heading);
+    const headingStyle = (renderedHeading === heading) ? 'code-block-heading' : 'code-block-heading-with-inline';
+      return '<div class="code-block">'
+      + `<div class="${headingStyle}"><span>${renderedHeading}</span></div>`
       + `<div class="code-block-content">${codeBlockContent}</div>`
       + '</div>';
```

Comment:
is there a better way than this?

should i just make a `.increased-line-height` class and apply it after I apply the base `.code-block-heading` class

Reply:
Could try this https://stackoverflow.com/questions/6078051/html-how-to-keep-spans-with-padding-and-borders-from-overlapping/38270183

Reply:
With
![image](https://user-images.githubusercontent.com/27397021/77227426-4215c200-6bbb-11ea-9da0-d0c850a828e6.png)

![image](https://user-images.githubusercontent.com/27397021/77227433-4cd05700-6bbb-11ea-8efb-792acd41f747.png)

The height of non-markdown headings have been increased as well. investigating further.

Reply:
It should be `.code-block-heading * { display: ... }`

Reply:
Hm, nevermind, i found an alternative.